packages/perseus/src/widgets/interactive-graphs/interactive-graph-question-builder.ts

```diff
@@ -91,10 +91,6 @@ class InteractiveGraphQuestionBuilder {
                         lockedFigures: this.lockedFigures,
                     },
                     type: "interactive-graph",
-                    version: {
-                        major: 0,
-                        minor: 0,
-                    },
```

Comment:
1. `version` is optional
2. Version 0.0.0 won't get upgraded

Reply:
Pedantically, widgets [default to v0.0.0](https://github.com/Khan/perseus/blob/7a984eba6f1cec3df314ec245d2176f5db190548/packages/perseus/src/widgets.ts#L232). Data missing a version also defaults to v0.0.0 and so does not need an upgrade.